### DISCUSSION TITLE
custom_config.h for my commonly played games

### DISCUSSION THREAD

**Discussion by LilRooks** [manna-harbour/miryoku#164] [Show and tell]
# [Link](https://github.com/LilRooks/miryoku_zmk/blob/master/miryoku/custom_config.h) (This post will be updated if it changes)

## Summary
- Miryoku Implementations
  - [ ] Miryoku QMK
  - [x] Miryoku ZMK
  - [ ] Miryoku KMonad
- Features
  - Layers
    - [ ] Substitute Layer
    - [x] Add Layer
    - [ ] Remove Layer
    - [ ] Change Layer Display Name
  - Change Mapping
    - [x] All Layers
    - [ ] Per Layer
  - Additional features
    - Mousekeys enabled
    - Gaming layer on left half minimizing most FPS rebinds (In my experience)
    - Right half of gaming layer is for mousekeys as well as some quick keys for games I play.
    - Diablo II layers similar to controller style layers in Resurrected.
    - Unused inner column repurposed to quick switch to Base, Diablo Base, and Gaming layers.
I am personally using the Technikable, so it is hard coded for that board.

**Comment by bryanprange** [comment]
[I'm also using a Technikable for gaming](https://github.com/manna-harbour/miryoku/discussions/158)

Good call using the inner column keys on the Technikable to toggle game layers, likely to steal that :smile:. Nice to avoid a combo when those keys are present and also unlikely to be hit accidentally.

Most of the FPS games that I play use the number row for selecting weapons. How is it having the number keys on the right side of the keyboard in your FPS gaming layer? Are you playing with mouse keys!?

Also noticed that you've got layer tap on your DONE layer thumb key, which is something I've been hesitant to introduce for my base game layer. Has that been predictable? Or do you find yourself hitting escape accidentally?

**Reply by LilRooks** [reply]
:o You're the other Technikable config I saw! I'm considering taking some things from your config as well, like the double tap boot, since the positioning of Alt-Tab on my gaming layer happens to be the same strokes for boot on base layer.

I probably should mention that I do have an MMO mouse (the kind with the numpad on the side), so the numbers are really only there in case i really need them, such as when i am doing some errands with the mousekeys over Bluetooth and I don't have the mouse.

The layer tap ESC is very useful, and I honestly put it there because I ran out of keys to put as an LT, but it's surprisingly consistent a long as I know i mean to go to that second layer... once I hold it, I'm committed. It's really intuitive for me coming from the controller scheme in D2R, and I may add something similar to my FPS layer.

**Reply by bryanprange** [reply]
:+1: awesome, yeah, the MMO mouse makes sense. I've never been good about using auxiliary mouse buttons, but, I'm starting to use them more to compensate for fewer keys - which is great.

My gaming layer has room for a layer toggle on the tertiary thumb which is used as ALT. That would be a good way to get the TGB, 890 keys available on 36 key boards. Might have to give it a shot!
